src/app.js

```diff
@@ -0,0 +1,82 @@
+import express from 'express';
+import logger from 'morgan';
+import cors from 'cors';
+import { initializeStatusManager } from './status.js';
+import issue from './issue.js'
+import revoke from './revoke.js'
+import generateSeed from './generate.js';
+import verifyAccess from './verifyAuthHeader.js'
+import { getConfig } from './config.js';
+
+const { enableStatusAllocation } = getConfig();
+
+export async function build(opts = {}) {
+
+    if (enableStatusAllocation) await initializeStatusManager()
+    
+    var app = express();
+
+    app.use(logger('dev'));
+    app.use(express.json());
+    app.use(express.urlencoded({ extended: false }));
+    app.use(cors())
+
+    app.get('/', function (req, res, next) {
+        res.send({ message: 'hello' })
+    });
+
+    app.post("/instance/:instanceId/credentials/issue", 
+        async (req, res) => {
+            try {
+                const instanceId = req.params.instanceId //the issuer instance/tenant with which to sign
+                try {
+                    await verifyAccess(req.headers.authorization, instanceId)
+                } catch (e) {
+                    return res.status(e.code).send(e.message)
+                }
+                const unSignedVC = req.body;
+                if (!req.body || !Object.keys(req.body).length ) return res.status(400).send('A verifiable credential must be provided in the body')
+                const signedVC = await issue(unSignedVC, instanceId)
+                return res.json(signedVC)
+            } catch (error) {
+                console.log(error);
+                return res.status(403).json(error);
+            }
+        })
+
+    // the body will look like:  {credentialId: '23kdr', credentialStatus: [{type: 'StatusList2021Credential', status: 'revoked'}]}
+    app.post("/instance/:instanceId/credentials/status", 
+        async (req, res) => {
+            try {
+                if (!enableStatusAllocation) return res.status(501).send("The status service has not been enabled.")
```

Comment:
This should be http status 503. (501 would be for features we didn't _implement_, vs just disabled by config)